tests/functional/common.config

```diff
@@ -72,6 +72,11 @@ export SED_PROG="`set_prog_path sed`"
 export BC_PROG="`set_prog_path bc`"
 [ "$BC_PROG" = "" ] && _fatal "bc not found"
 
+PGREP_OPTIONS='-a'
+if pgrep * -a 2>&1 | grep -q "invalid option";then
```

Comment:
just updating your branch with `git push -f` is enough for this case. If you want to avoid `push -f`, closing an existing PR and open a new PR based on new branch will be good.